### ISSUE TITLE
 Got audio2huff.py working?

### ISSUE THREAD

**Issue by sesselastronaut** [Roger-random/mozzi_wilhelm#1]
This is more a question than an issue: was wondering if and how you got audio2huff.py working. I struggle to get scikits.audiolab installed on my current setup (Debian testing). Issues are reported here in 2018: [https://github.com/sensorium/Mozzi/issues/54](https://github.com/sensorium/Mozzi/issues/54)

**Comment by Roger-random** [comment]
The comment block at the beginning of ```audio2huff.py``` listed its dependencies and URLs to each of them. I recall I was able to follow each of their installation instructions to get everything running.

The only problem I encountered were mixups between Python 2 and 3. Once I was running exclusively on Python 2 I was fine.

https://newscrewdriver.com/2020/01/14/arduino-mozzi-space-core-exercise/

Hope this helps!
